rest/taskana-rest-spring/src/test/java/pro/taskana/task/rest/TaskControllerIntTest.java

```diff
@@ -1395,6 +1392,62 @@ void should_ThrowException_When_CancelClaimingOfClaimedTaskByAnotherUser() {
         .isEqualTo(HttpStatus.BAD_REQUEST);
   }
 
+  @Test
+  void should_RequestReviewTask() {
+    String url =
+        restHelper.toUrl(RestEndpoints.URL_TASKS_ID, "TKI:000000000000000000000000000000000035");
+    HttpEntity<Object> auth = new HttpEntity<>(RestHelper.generateHeadersForUser("user-1-1"));
+
+    // retrieve task from Rest Api
+    ResponseEntity<TaskRepresentationModel> getTaskResponse =
+        TEMPLATE.exchange(url, HttpMethod.GET, auth, TASK_MODEL_TYPE);
+    assertThat(getTaskResponse.getBody()).isNotNull();
+    TaskRepresentationModel claimedTaskRepresentationModel = getTaskResponse.getBody();
+    assertThat(claimedTaskRepresentationModel.getState()).isEqualTo(TaskState.CLAIMED);
+    assertThat(claimedTaskRepresentationModel.getOwner()).isEqualTo("user-1-1");
+
+    // request review
+    String url2 =
+        restHelper.toUrl(
+            RestEndpoints.URL_TASKS_ID_REQUEST_REVIEW, "TKI:000000000000000000000000000000000035");
+    ResponseEntity<TaskRepresentationModel> cancelClaimResponse =
```

Comment:
Please name the variable cancelClaimResponse according. E. g. requestReviewResponse